README.md

```diff
@@ -0,0 +1,24 @@
+# Getting Started
```

Comment:
Try to add a small readme with your endpoints and basic auth users.

Reply:
I will add more information in the swagger